### DISCUSSION TITLE
Episode 178: Henry the Clock, chunk_by & more!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#70] [Episode]
Episode Link: https://adspthepodcast.com/2024/04/19/Episode-178.html

Feel free to write comments or ask questions 🙂

**Comment by Ignition** [comment]
I was at that ACCUConf talk. It was excellent. Started off with the parallel scan that I knew, and then straight into a deep dive into all the extra implementation details. Monotonic property and decoupled look back are valuable insights I learnt.

You didn't need to worry about losing your audience. You did great explaining the same concept multiple times in multiple forms. You explained, then reinforced with visual walk-through, the ratcheted up the complexity with more complex application. The more complex examples used the same approach with additional insights and refinement. Building and reinforcing what you did in earlier parts of the talk.

All to say, you are a great explainer of a complex subject. Either natural understanding of pedagogy, or at least went to great effort to be good at it.

(Also appreciated applauding your mum at the start of the talk)
